# Translation of Odoo Server.
# This file contains the translation of the following modules:
# * account_plaid
# 
# Translators:
# Connie Xiao <connie.xiao@elico-corp.com>, 2015
# fausthuang, 2015
# Jeffery Chenn <jeffery9@gmail.com>, 2016
msgid ""
msgstr ""
"Project-Id-Version: Odoo 9.0\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-10-01 06:30+0000\n"
"PO-Revision-Date: 2016-06-16 05:34+0000\n"
"Last-Translator: Jeffery Chenn <jeffery9@gmail.com>\n"
"Language-Team: Chinese (China) (http://www.transifex.com/odoo/odoo-9/language/zh_CN/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Language: zh_CN\n"
"Plural-Forms: nplurals=1; plural=0;\n"

#. module: account_plaid
#: model:ir.model.fields,field_description:account_plaid.field_online_account_token
msgid "Access Token"
msgstr "存取令牌"

#. module: account_plaid
#: code:addons/account_plaid/models/plaid.py:52
#: code:addons/account_plaid/models/plaid.py:63
#, python-format
msgid "An error has occurred while trying to connect to plaid service"
msgstr "当连接plaid 服务时, 报错"

#. module: account_plaid
#: model:ir.model,name:account_plaid.model_online_account
msgid "Email Thread"
msgstr "邮件链"

#. module: account_plaid
#: code:addons/account_plaid/models/plaid.py:129
#, python-format
msgid "Error in synchronization"
msgstr "同步时报错"

#. module: account_plaid
#: code:addons/account_plaid/models/plaid.py:136
#, python-format
msgid "Error with Plaid.com"
msgstr "Plaid.com 错误"

#. module: account_plaid
#: model:ir.model,name:account_plaid.model_account_journal
msgid "Journal"
msgstr "日记账"

#. module: account_plaid
#: model:ir.model.fields,field_description:account_plaid.field_account_journal_onlinesync_config_plaid_account_id
msgid "Online Account"
msgstr "在线账户"

#. module: account_plaid
#: model:ir.model.fields,field_description:account_plaid.field_online_account_plaid_id
msgid "Plaid Account"
msgstr "plaid 账户"

#. module: account_plaid
#: model:ir.ui.view,arch_db:account_plaid.account_config_settings_view_inherit_plaid
msgid "Plaid Configuration"
msgstr "plaid 配置"

#. module: account_plaid
#: model:ir.model.fields,field_description:account_plaid.field_account_config_settings_plaid_id
msgid "Plaid ID"
msgstr "Plaid ID"

#. module: account_plaid
#: model:ir.model.fields,field_description:account_plaid.field_account_config_settings_plaid_secret
msgid "Plaid Secret"
msgstr "plaid 机密"

#. module: account_plaid
#: code:addons/account_plaid/models/plaid.py:130
#, python-format
msgid ""
"The synchronization of the journal %s with the plaid account %s has "
"failed.<br>The error message is :<br>%s"
msgstr "plaid 同步账户%s 和科目 %s 不成功.<br>错误信息为 :<br>%s"

#. module: account_plaid
#: code:addons/account_plaid/models/plaid.py:137
#, python-format
msgid ""
"The synchronization with Plaid.com failed. Please check the error : <br> %s"
msgstr "plaid 同步失败. 请检查错误信息 : <br> %s"

#. module: account_plaid
#: code:addons/account_plaid/models/plaid.py:38
#, python-format
msgid ""
"You haven't configure your plaid account, please go to accounting/settings "
"to configure it"
msgstr "你没有设置plaid 科目, 请去财务/设置下去定义"

#. module: account_plaid
#: model:ir.model,name:account_plaid.model_account_config_settings
msgid "account.config.settings"
msgstr "会计.配置.设定"

#. module: account_plaid
#: model:ir.model,name:account_plaid.model_account_journal_onlinesync_config
msgid "account.journal.onlinesync.config"
msgstr "科目.账.在线同步.配置"

#. module: account_plaid
#: model:ir.model,name:account_plaid.model_online_institution
msgid "online.institution"
msgstr "在线.指示"
